Dockerfile

```diff
@@ -0,0 +1,15 @@
+# golang:1.17 is based on debian:11, this is important to ensure we have libc compatibility for the copied binary
+
+FROM docker.io/library/golang:1.17 as builder
+ENV CGO_ENABLED=1
+COPY . /usr/src/mantle
+RUN cd /usr/src/mantle && ./build
+
+FROM docker.io/library/debian:11
+RUN apt-get update && apt-get upgrade -y && apt-get install --no-install-recommends -y qemu-utils qemu-system-x86 qemu-system-aarch64 qemu-efi-aarch64 seabios ovmf lbzip2 sudo dnsmasq gnupg2 git curl iptables nftables dns-root-data ca-certificates sqlite3
+COPY --from=builder /usr/src/mantle/bin /usr/local/bin
+RUN printf '#!/bin/sh\n# Workaround for kola to find its kola binary, stored in sbin to get precedence\nexec /usr/local/bin/kola "$@"\n' > /usr/local/sbin/kola
```

Comment:
Can you explain what is happening here?

Reply:
Oh, it should be `to find its kolet binaries`. Didn't check how kola's equivalent to finding the path from `$0` is implemented but it somehow can't cope with being called by `kola` in another directory, only with the full path.

Reply:
Updated the explaination.

From a quick look kola uses `filepath.Dir(os.Args[0])` or `/usr/lib/kola`, so it either need to be called with a full or relative path or we move the kolet binaries to `/usr/lib/kola`. I think it would be good to improve the logic in kola directly as this came up sometimes already.

Reply:
Ok, now I understand.

Reply:
There is https://pkg.go.dev/os#Executable which we can use like this:

```
	p, err := os.Executable()
	if err != nil {
		if (strings.Contains(os.Args[0], "/")) {
			p =  os.Args[0]
		} else {
			p, err = exec.LookPath(os.Args[0])
			if err != nil {
			p =  os.Args[0]
			}
		}
	}
	fmt.Println(filepath.Dir(p))
```

Reply:
Will create a follow-up PR